backend/src/endpoints/v1/auth/jwt_auth.py

```diff
@@ -0,0 +1,75 @@
+from fastapi import APIRouter, Depends
+from fastapi.security import OAuth2PasswordRequestForm
+
+from src.core.deps import get_user_from_access_token, get_user_service
+from src.core.schemas import UserRead, UserRegister
+from src.core.schemas.token.base import TokenInfo
+from src.services import SecurityService, UserService
+from src.utils.security.jwt_token import jwt_token
+
+router = APIRouter(prefix="/auth", tags=["Auth"])
+
+
+@router.post(
+    "/signup",
+    response_model=UserRead,
+    response_model_exclude_none=True,
+    responses={
+        400: {
+            "description": "Bad Request",
+            "content": {
+                "application/json": {
+                    "example": {"detail": "Failed to create user"},
+                },
+            },
+        },
+        409: {
+            "description": "Registration Error",
+            "content": {
+                "application/json": {
+                    "example": {"detail": "User already exists"},
+                },
+            },
+        },
```

Comment:
I'm not a big fun of doing it in the `endpoints` file. I would prefer to make additional folder for OpenAPI documentation. OPTIONAL

Reply:
It takes lot's of space

Reply:
Removed